maintenance/wikia/sql/dataware-schema.sql

```diff
@@ -14,562 +14,1046 @@
+-- Table structure for table `video_migration_log`
 --
 
-SET @saved_cs_client     = @@character_set_client;
-SET character_set_client = utf8;
-CREATE TABLE IF NOT EXISTS `user_history` (
-  `user_id` int(5) unsigned NOT NULL,
-  `user_name` varchar(255) character set latin1 collate latin1_bin NOT NULL default '',
-  `user_real_name` varchar(255) character set latin1 collate latin1_bin NOT NULL default '',
-  `user_password` tinyblob NOT NULL,
-  `user_newpassword` tinyblob NOT NULL,
-  `user_email` tinytext NOT NULL,
-  `user_options` blob NOT NULL,
-  `user_touched` varchar(14) character set latin1 collate latin1_bin NOT NULL default '',
-  `user_token` varchar(32) character set latin1 collate latin1_bin NOT NULL default '',
-  `uh_timestamp` timestamp NOT NULL default CURRENT_TIMESTAMP,
-  KEY `user_name` (`user_name`(10)),
-  KEY `idx_user_history_timestamp` (`uh_timestamp`),
-  KEY `idx_user_id` (`user_id`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8;
-SET character_set_client = @saved_cs_client;
+DROP TABLE IF EXISTS `video_migration_log`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `video_migration_log` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `action_time` char(14) NOT NULL,
+  `wiki_id` int(11) NOT NULL,
+  `action_name` varchar(255) NOT NULL,
+  `action_status` varchar(255) NOT NULL,
+  `action_desc` varchar(255) NOT NULL,
+  `wiki_dbname` varchar(255) DEFAULT NULL,
+  PRIMARY KEY (`id`),
+  KEY `city_id` (`wiki_id`,`action_time`)
+) ENGINE=InnoDB AUTO_INCREMENT=989939 DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
 
 --
--- Table structure for table `user_login_history`
+-- Table structure for table `video_migration_sanitization`
 --
 
-SET @saved_cs_client     = @@character_set_client;
-SET character_set_client = utf8;
-CREATE TABLE IF NOT EXISTS `user_login_history` (
-  `user_id` int(5) unsigned NOT NULL,
-  `city_id` int(9) unsigned default '0',
-  `ulh_timestamp` timestamp NOT NULL default CURRENT_TIMESTAMP,
-  `ulh_from` tinyint(4) default '0',
-  `ulh_rememberme` tinyint(4) default NULL,
-  KEY `idx_user_login_history_timestamp` (`ulh_timestamp`),
-  KEY `idx_user_id` (`user_id`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8;
-SET character_set_client = @saved_cs_client;
+DROP TABLE IF EXISTS `video_migration_sanitization`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `video_migration_sanitization` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `city_id` int(11) NOT NULL,
+  `old_title` varchar(255) NOT NULL,
+  `sanitized_title` varchar(255) NOT NULL,
+  `operation_status` char(14) NOT NULL DEFAULT 'UNKNOWN',
+  `operation_time` char(14) NOT NULL,
+  `article_title` varchar(255) NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `city_id` (`city_id`,`old_title`),
+  KEY `sanitized_title` (`sanitized_title`)
```

Comment:
@garthwebb: do we need this table? It look like a leftover after videos migration.